Fix bpo-29528 Use a secure variable to stop spam

dstufft opened the pull request:
If the IRC notification is stored in plaintext, then anyone who forks the repository and also adds it to travis will send notifications to the IRC channel for their fork by default. Since the secure variable is encrypted using a repository specific key, this will only work when it is being built using the correct repository.

dianaclarke commented:
This is a read-only mirror of cpython. Please see the devguide (https://docs.python.org/devguide/index.html) for more information about contributing.

dstufft committed:
Fix bpo-29528 Use a secure variable to stop spam

If the IRC notification is stored in plaintext, then anyone who forks
the repository and also adds it to travis will send notifications to
the IRC channel for their fork by default. Since the secure variable
is encrypted using a repository specific key, this will only work when
it is being built using the correct repository.
